griptape/config/config.py

```diff
@@ -6,10 +6,21 @@
 from .openai_driver_config import OpenAiDriverConfig
 
 
-@define
+@define(kw_only=True)
 class _Config(BaseConfig):
-    drivers: BaseDriverConfig = field(default=Factory(lambda: OpenAiDriverConfig()), kw_only=True)
-    logging: LoggingConfig = field(default=Factory(lambda: LoggingConfig()), kw_only=True)
+    logging: LoggingConfig = field(default=Factory(lambda: LoggingConfig()))
+    _drivers: BaseDriverConfig = field(default=None)
```

Comment:
[fyi on aliases (aliai)](https://www.attrs.org/en/stable/init.html#private-attributes-and-aliases)
> attrs follows the doctrine that [there is no such thing as a private argument](https://github.com/hynek/characteristic/issues/6) and strips the underscores from the name when writing the __init__ method signature


so removing `alias=drivers` makes no difference and its just less obvious